cypress/fixtures/hello_view.json

```
@@ -0,0 +1,57 @@
+{
+  "assemblies": [
+    {
+      "name": "volvox",
+      "aliases": ["vvx"],
+      "sequence": {
+        "type": "ReferenceSequenceTrack",
+        "trackId": "volvox_refseq",
+        "adapter": {
+          "type": "TwoBitAdapter",
+          "twoBitLocation": {
+            "uri": "test_data/volvox/volvox.2bit"
+          }
+        }
+      },
+      "refNameAliases": {
+        "adapter": {
+          "type": "FromConfigAdapter",
+          "features": [
+            {
+              "refName": "ctgA",
+              "uniqueId": "alias1",
+              "aliases": ["A", "contigA"]
+            },
+            {
+              "refName": "ctgB",
+              "uniqueId": "alias2",
+              "aliases": ["B", "contigB"]
+            }
+          ]
+        }
+      }
+    }
+  ],
+  "plugins": [
+    {
+      "name": "MyProject",
+      "url": "http://localhost:9000/dist/jb2-plugin-testing.umd.development.js"
+    }
+  ],
+  "defaultSession": {
+    "id": "j8wx-zukm",
+    "name": "New session 1/12/2021, 12:32:05 PM",
+    "margin": 0,
+    "drawerWidth": 384,
+    "views": [
+      {
+        "type": "HelloView"
+      }
+    ],
+    "widgets": {},
+    "activeWidgets": {},
+    "connectionInstances": {},
+    "sessionTracks": [],
+    "sessionConnections": []
+  }
```

This can be simplified

```suggestion
  "defaultSession": {
    "name": "New session",
    "views": [
      {
        "type": "HelloView"
      }
    ],
  }
```

---

Fixed